### PR TITLE
Save theme to localstorage (fixes #15)

### LINKED ISSUE #15: Dark mode doesn't get saved
### Steps to reproduce:
- Select dark mode
- Hard-refresh the page (F5 or Ctrl+R, or ...)

#### Actual behaviour: the application switches to light mode

### PR DESCRIPTION
**Pull request by Yakimych**

### CODE REVIEW

File: src/Root.re [hunk 1]
```diff
@@ -1,14 +1,43 @@
+open Dom.Storage2;
+open ThemeContext;
+
+let themeStorageKey = "theme";
+
+let themeToString =
```
Comment (MargaretKrutikova) [inline src/Root.re]: I think it would be better to collocate those functions with the type `theme` from `ThemeContext`, what do you think?

**Comment by MargaretKrutikova** [comment]
Thanks a lot for the fix! Left just one small comment 😄

File: src/Root.re (hunk 1 above)
Comment (Yakimych) [reply src/Root.re]: Sure. I actually started with them there, but moved them here later for some reason (probably in order to not change more than one file in the PR).
Moved them back to ThemeContext now.

**Comment by Yakimych** [comment]
No problem. It's always enjoyable to work with Reason 😁 
Thanks for the app - it really cool!

**Pull request merged**